dsl-reference.md

```diff
@@ -588,13 +588,13 @@ Allows workflows to iterate over a collection of items, executing a defined set
 
 ##### Properties
 
-| Name | Type | Required | Description|
-|:--|:---:|:---:|:---|
-| for.each | `string` | `no` | The name of the variable used to store the current item being enumerated.<br>Defaults to `item`. |
-| for.in | `string` | `yes` | A [runtime expression](dsl.md#runtime-expressions) used to get the collection to enumerate. |
-| for.at | `string` | `no` | The name of the variable used to store the index of the current item being enumerated.<br>Defaults to `index`. |
-| while | `string` | `no` | A [runtime expression](dsl.md#runtime-expressions) that represents the condition, if any, that must be met for the iteration to continue. |
-| do | [`task`](#task) | `yes` | The task to perform for each item in the collection. |
+| Name |             Type             | Required | Description|
```

Comment:
Can you please remove whitespaces/tabs?

Reply:
yes - it was added automatically by my IDE, not sure why